packages/@tinacms/graphql/src/database/index.ts

```diff
@@ -160,41 +213,47 @@ export class Database {
         payload = data
       }
       const extension = path.extname(filepath)
-      const stringData = this.stringifyFile(
+      const stringifiedFile = stringifyFile(
         payload,
         extension,
         templateInfo.type === 'union'
       )
-      await this.bridge.put(filepath, stringData)
+      return { stringifiedFile, payload }
     }
-    return true
+  }
+
+  public flush = async (filepath: string) => {
```

Comment:
Currently this is only used for testing purposes, it returns what we _would_ write the `bridge`, but doesn't actually write it.